alcs-frontend/src/app/features/home/home.component.ts

```diff
@@ -9,17 +9,26 @@ import { AuthenticationService, ICurrentUser } from '../../services/authenticati
 export class HomeComponent implements OnInit {
   currentUser!: ICurrentUser;
   hasGIS = false;
+  hasCommissioner = false;
 
   constructor(private authService: AuthenticationService) {}
 
   ngOnInit(): void {
     this.currentUser = this.authService.getCurrentUser()!;
 
-    this.hasGIS = this.currentUser.client_roles ? this.currentUser.client_roles.includes('GIS') : false;
-
     this.authService.$currentUser.subscribe((currentUser) => {
       if (currentUser) {
         this.currentUser = currentUser;
+
+        this.hasGIS =
+          currentUser.client_roles && currentUser.client_roles.length === 1
+            ? currentUser.client_roles.includes(ROLES.GIS)
```

Comment:
does this mean that users with only 1 role = GIS will be able to get hasGIS?

Reply:
This is to check if the are ONLY GIS, since the current requirements you should only see one thing on your home page.